README.md

```diff
@@ -6,29 +6,31 @@ carbonite-migrate
 User requirements
 =================
 
-**The following software must be running/installed on the client computer**:
+**The following software must be running/installed on the user's computer**:
```

Comment:
Maybe we should reword this since it makes it seem like Device42 needs to be installed on the user's computer.  Maybe we should just say:
The following software must be installed and running:

Reply:
Sure, done.